pkg/build/api/validation/validation_test.go

```diff
@@ -1269,7 +1269,7 @@ func TestValidateCommonSpec(t *testing.T) {
 				Output: buildapi.BuildOutput{
 					To: &kapi.ObjectReference{
 						Kind: "DockerImage",
-						Name: "some/long/value/with/no/meaning",
+						Name: "index.docker.io/some/long/value/with/no/meaning",
```

Comment:
Ah, right, at least for the hub. Although as I wrote in another comment, if/when the hub allows `[user]/[one]/[or]/[more]/[segments]`, these tests won't be valid any more